config/seata-config-spring-cloud/pom.xml

```diff
@@ -18,8 +18,9 @@
             <version>${project.parent.version}</version>
         </dependency>
         <dependency>
-            <groupId>org.springframework</groupId>
-            <artifactId>spring-context</artifactId>
+            <groupId>io.seata</groupId>
+            <artifactId>seata-spring</artifactId>
+            <version>${project.parent.version}</version>
```

Comment:
seata-spring depends on tm, rm, rm-datasource, etc. It is recommended to exclude all.

Reply:
OK.

Reply:
fixed.